src/main/java/teammates/client/scripts/DataMigrationForSanitizedData.java

```diff
@@ -0,0 +1,85 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Level;
+
+import javax.jdo.PersistenceManager;
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.StringHelper;
+import teammates.common.util.Utils;
+import teammates.logic.core.StudentsLogic;
+import teammates.storage.datastore.Datastore;
+
+public class DataMigrationForSanitizedData extends RemoteApiClient {
+
+    public static void main(String[] args) throws IOException {
+        DataMigrationForSanitizedData migrator = new DataMigrationForSanitizedData();
+        migrator.doOperationRemotely();
```

Comment:
Could you add a boolean variable for previewing the modifications of the script? So that before running the data migration, we can do a verification of the changes.